src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java

```diff
@@ -101,8 +97,8 @@ private void uploadNewDocumentVersion(File file, CreateTemplateRenditionDto crea
                     .Builder()
                     .setType(MultipartBody.FORM)
                     .addFormDataPart("file", createTemplateRenditionDto.getFullOutputFilename(),
-                            RequestBody.create(MediaType.get(createTemplateRenditionDto.getOutputType().getMediaType()),
-                                    file))
```

Comment:
this is a bug?

Reply:
SonarQube was complaining that the create method is deprecated ("@Deprecated" code should not be used - 
java:S1874). I googled it and apparently swapping the order of the arguments fixes it - https://newbedev.com/how-to-replace-deprecated-okhttp-requestbody-create